odml/property.py

```diff
@@ -587,9 +587,12 @@ def extend(self, obj, strict=True):
             return
 
         new_value = self._convert_value_input(obj)
+        special_dtypes = ["url", "person", "text"]
```

Comment:
Please move to `dtypes.py` and use it from there so that it is defined only once and tied to `dtypes`.